localization_manager/config/parameters.yaml

```diff
@@ -40,7 +40,7 @@ sequential_timesteps_until_gps_operation: 5
 # 3 - AUTO select between NDT and GNSS without GNSS timeout
 # 4 - GNSS only with NDT initialization
 # 5 - GNSS only with fixed offset
-localization_mode: 4
+localization_mode: 5
```

Comment:
Probably still aren't ready to make this the default since it requires editing the map offset locally.